v2/jdbc-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/JdbcToBigQueryIT.java

```diff
@@ -191,6 +191,34 @@ public void testPostgresToBigQueryFlex() throws IOException {
                     + testName));
   }
 
+  @Test
+  public void testPostgresWithUnicodeCharactersInQuery() throws IOException {
+    String tableName = "unicóde_table";
```

Comment:
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/bigquery/BigQueryResourceManagerUtils.java#L95-L100

BigQuery ILLEGAL_TABLE_CHARS regex needs to be modified to either allow unicode characters according to https://cloud.google.com/bigquery/docs/tables or disable this part of the checkValidTableId entirely and allow the test to fail when it sends create table request to BQ

Reply:
Fixed. Let's see if the IT succeeds now.